context/app/static/js/pages/Genes/Genes.jsx

```diff
@@ -25,7 +29,7 @@ function Genes() {
           Gene
         </Typography>
         <Typography variant="h1" component="h2">
```

Comment:
I guess this was in the scope of the last PR rather than this one, but I'm a bit skeptical of the HTML semantics here (lines 24 and 27); making `Gene` the top level heading of the page (i.e. making it an `h1`) while making the gene's common name an `h2` seems a bit like making `Blog` the top level heading of a specific blog page rather than the title of the article. Maybe this should be reversed? (i.e. with Gene having `component="h2"` and commonName having `component="h1"`)

Reply:
I agree with this.